src/mds/MDLog.cc

```diff
@@ -992,8 +992,9 @@ void MDLog::_recovery_thread(MDSInternalContextBase *completion)
   }
 
   /* Read the header from the front journal */
-  Journaler *front_journal = new Journaler(jp.front, mds->mdsmap->get_metadata_pool(),
-      CEPH_FS_ONDISK_MAGIC, mds->objecter, logger, l_mdl_jlat, &mds->timer, mds->finisher);
+  Journaler *front_journal = new Journaler("mdlog", jp.front,
```

Comment:
These Journaler instances are (depending on whether doing reformat) later assigned to the main MDLog::journaler, so special names would not make sense any more at that point